app/models/card.rb

```diff
@@ -1,2 +1,10 @@
 class Card < ActiveRecord::Base
+  validates_presence_of :original_text, :translated_text, :review_date
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.